Frostblademaster - ai no miko quests + the rewards script

Hassan0Kamel opened the pull request:
my first ever scripts so hopefully I get constructive criticism, both scripts are seasonal btw.

Hassan0Kamel committed:
Frostblademaster - ai no miko quests + the rewards script

my first ever scripts so hopefully I get constructive criticism, both scripts are seasonal btw.

Hassan0Kamel committed:
Update frostblademaster- ai no miko.cs

fixed a little error that I encountered (adding this in Core.PreLoad())
now it compiles well

Hassan0Kamel committed:
Update Search For Gifts Rewards.cs

did  include the appropriate story script so if someone didn't do the storyline and aimed for the rewards directly, it will do the story first.

wtffidy commented on Seasonal/Frostvale/Search For Gifts Rewards.cs:
```diff
@@ -0,0 +1,37 @@
+//cs_include Scripts/CoreBots.cs
+using Skua.Core.Interfaces;
+
+public class SearchForGiftsRewards
```
public class can not be the same as the public void name

wtffidy commented on Seasonal/Frostvale/Search For Gifts Rewards.cs:
```diff
@@ -0,0 +1,37 @@
+
+        Core.AddDrop(rewards);
+
+        Core.RegisterQuest(6992);
```
RegisterQuest > RegisterQuests

Hassan0Kamel committed:
Update Search For Gifts Rewards.cs

done resolving the required 2 edits, one on register quests function and the other was the public class and public void names

Hassan0Kamel committed:
Update and rename Search For Gifts Rewards.cs to SearchForGiftsRewards.cs

done removing spaces from the name and from the inclusion of the story script

Hassan0Kamel committed:
Rename frostblademaster- ai no miko.cs to frostblademaster.cs

done removing the space from the name
